cmd/query/app/querysvc/adjuster/ipattribute.go

```diff
@@ -20,7 +20,7 @@ var ipAttributesToCorrect = map[string]struct{}{
 // IPAttribute returns an adjuster that replaces numeric "ip" attributes,
 // which usually contain IPv4 packed into uint32, with their string
 // representation (e.g. "8.8.8.8"").
-func IPAttribute() IPAttributeAdjuster {
+func IPAttribute() Adjuster {
```

Comment:
I think this was fine before. It's ok to return structs from constructor instead of interface, my comment was about when using Func() to return an interface since Func() is not a struct.